pom.xml

```diff
@@ -1319,8 +1319,8 @@
     <powermockVersion>1.6.4</powermockVersion>
     <okhttpVersion>3.10.0</okhttpVersion>
     <okioVersion>1.14.0</okioVersion>
-    <opennmsApiVersion>0.4.1</opennmsApiVersion>
-    <opennmsApiVersionOsgi>0.4.1</opennmsApiVersionOsgi>
+    <opennmsApiVersion>0.4.2-SNAPSHOT</opennmsApiVersion>
```

Comment:
This forces to  make a release of  integration api before we make next point release. Just adding as a note.